docs/getting-started.md

```diff
@@ -238,7 +245,7 @@ Setting up your development environment can be somewhat tedious if you're new to
 
 <h4>1. Install Android Studio</h4>
 
-[Download and install Android Studio](https://developer.android.com/studio/index.html). Choose a "Custom" setup when prompted to select an installation type. Make sure the boxes next to all of the following are checked:
+[Download and install Android Studio](https://developer.android.com/studio/index.html). Through the Android Studio Installation Wizard, make sure to check these itens to be installed:
```

Comment:
Typo -`itens`.

Reply:
What about 'While on Android Studio intallation wizard , make sure the boxes next to all of the following items are checked'?

Reply:
The proposed sentence sounds better for me, but I'm not a native English speaker. 

Please update it and I'm going to ask @rachelnabors for the review 😄